README.md

```diff
@@ -1,193 +1,266 @@
+## Getting help or getting involved
+Catalyst developers are active on the [Julia Discourse](https://discourse.julialang.org/), 
+the [Julia Slack](https://julialang.slack.com) channels \#sciml-bridged and \#sciml-sysbio, and the 
+[Julia Zulip sciml-bridged channel](https://julialang.zulipchat.com/#narrow/stream/279055-sciml-bridged). 
+For bugs or feature requests, [open an issue](https://github.com/SciML/Catalyst.jl/issues).
 
+If you are interested in participating in the development of Catalyst, or integrating your package(s) 
+with it, developer documentation can be found [here](@ref ref). Are you a student (or similar) who 
+wishes to do a Google Summer of Code (or similar) project tied to Catalyst? Information on how to get 
+involved, including good first issues to get familiar with working on the package, can be found [here](@ref ref).
 
 ## Supporting and citing Catalyst.jl
-The software in this ecosystem was developed as part of academic research. If you would like to help support it,
-please star the repository as such metrics may help us secure funding in the future. If you use Catalyst as part
-of your research, teaching, or other activities, we would be grateful if you could cite our work:
+The software in this ecosystem was developed as part of academic research. If you would like to help 
+support it, please star the repository as such metrics may help us secure funding in the future. If 
+you use Catalyst as part of your research, teaching, or other activities, we would be grateful if you 
+could cite our work:
 ```
 @article{CatalystPLOSCompBio2023,
-    doi = {10.1371/journal.pcbi.1011530},
-    author = {Loman, Torkel E. AND Ma, Yingbo AND Ilin, Vasily AND Gowda, Shashi AND Korsbo, Niklas AND Yewale, Nikhil AND Rackauckas, Chris AND Isaacson, Samuel A.},
-    journal = {PLOS Computational Biology},
-    publisher = {Public Library of Science},
-    title = {Catalyst: Fast and flexible modeling of reaction networks},
-    year = {2023},
-    month = {10},
-    volume = {19},
-    url = {https://doi.org/10.1371/journal.pcbi.1011530},
-    pages = {1-19},
-    number = {10},
+ doi = {10.1371/journal.pcbi.1011530},
+ author = {Loman, Torkel E. AND Ma, Yingbo AND Ilin, Vasily AND Gowda, Shashi AND Korsbo, Niklas AND Yewale, Nikhil AND Rackauckas, Chris AND Isaacson, Samuel A.},
+ journal = {PLOS Computational Biology},
+ publisher = {Public Library of Science},
+ title = {Catalyst: Fast and flexible modeling of reaction networks},
+ year = {2023},
+ month = {10},
+ volume = {19},
+ url = {https://doi.org/10.1371/journal.pcbi.1011530},
+ pages = {1-19},
+ number = {10},
 }
 ```
+
+We also maintain a user survey, asking basic questions about how users utilise the package. The survey 
+is available [here](ref), and only takes about 5 minutes to fill out. We are grateful to those who 
+fill out the survey, as this helps us further develop the package.
```

Comment:
Again, creating the survey should be rather simply, but we have other stuff to do before then.I will remove it after you have read (again I put it here when I thought we might have the survey ready, and now I keep it until you have read, just so you know it is an idea of mine to put something like this here)